client/my-sites/checkout/checkout/secure-payment-form.jsx

```diff
@@ -542,13 +540,8 @@ export class SecurePaymentForm extends Component {
 	renderGreatChoiceHeader() {
 		const { translate } = this.props;
 		const headerText = translate( 'Great choice! How would you like to pay?' );
-
-		if ( 'variantRightColumn' === abtest( 'showCheckoutCartRight' ) ) {
-			const element = document.getElementsByClassName( 'formatted-header__title' )[ 0 ];
-			element.textContent = headerText;
-			return;
-		}
-		return <FormattedHeader headerText={ headerText } />;
+		const element = document.getElementsByClassName( 'formatted-header__title' )[ 0 ];
+		element.textContent = headerText;
```

Comment:
Could you add a bit of info why we do it this way instead of the React way?

Reply:
You're right @dzver , this needs to change. I've implemented the React way in [3b4b80e](https://github.com/Automattic/wp-calypso/pull/33560/commits/0d61e898455e789f98ecebf359624a01fa259803).